frontend/src/features/frame/components/sidebar/SideBar.tsx

```diff
@@ -91,49 +92,67 @@ const useStyles = makeStyles((theme: Theme) => ({
 }));
 
 const SideBar = () => {
-  const { t } = useTranslation();
   const classes = useStyles();
-  const drawerOpen = useAppSelector(selectFrame);
-  const dispatch = useAppDispatch();
   const theme = useTheme();
-  const isSmallScreen = useMediaQuery(theme.breakpoints.down('md'));
+
+  const { t } = useTranslation();
   const location = useLocation();
+  const dispatch = useAppDispatch();
+
+  const { options } = useCurrentPoll();
+  const path = options && options.path ? options.path : '/';
+  const disabledItems =
+    options && options.disabledMenuItems ? options.disabledMenuItems : [];
```

Comment:
(minor) can be written more simply with optional chaining:
```suggestion
  const disabledItems = options?.disabledMenuItems ?? [];
```